extensions/ql-vscode/src/databases/config/db-config.ts

```diff
@@ -88,6 +90,7 @@ export interface LocalDatabase {
   name: string;
   dateAdded: number;
   language: string;
+  source: DatabaseSource;
```

Comment:
Is the idea that this doesn't need to be an optional field because the UI currently doesn't allow you to add local databases to this config? This is the config for the new, combined databases panel, right?

Reply:
Yes, exactly. This type isn't actually used anywhere, but once we start using it, we should have the source for every database that is added. This is the same as the `language` and `dateAdded`, both of which are also optional in the `FullDatabaseOptions`, but required in this type.